idseq/uploader.py

```diff
@@ -321,6 +324,12 @@ def validate_project(base_url, headers, project_name):
     return project_name, names_to_ids[project_name]
 
 
+def get_match_in_dict(keys, dictionary):
+    for k in keys:
+        if k in dictionary:
+            return dictionary[k]
```

Comment:
Could we pop this value from the dictionary, so we don't send `host_genome` as metadata as well?